backend/app/assets/stylesheets/spree/backend/shared/_layout.scss

```diff
@@ -129,3 +129,11 @@ body {
   border-top: 1px solid $color-border;
   padding: 10px 0;
 }
+
+// Button Spacing
```

Comment:
If we need comments like these we know we're putting this in the wrong place. I would prefer to see these styles in a file like `_forms.scss`.

What these styles do need, though, is a comment explaining their purpose.